### PR TITLE
Sign installer in bamboo

### LINKED ISSUE #457: Installer: Should be signed
If you are first time installing MIES on a windows 10 box you get a scary "This is unsafe" warning. This can be prevented by signing the installer.

Discussion:
- http://forums.winamp.com/showthread.php?t=395644
- http://forums.winamp.com/showthread.php?t=455491
- https://docs.microsoft.com/en-us/windows/msix/package/sign-app-package-using-signtool

We need a certificate for that (either the AI buys one or I buy one). It costs around 500 $USD per year. See also https://www.digicert.com/code-signing/.

### PR DESCRIPTION
**Pull request by t-b**
- [x] Get it working remotely
- [x] Add support in bamboo

Close #457.

### CODE REVIEW

File: tools/sign-installer.sh [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+# - https://sectigo.com/faqs/detail/EV-Code-Signing-Certificates-Collection/kA01N000000brbF
+# - https://www.reddit.com/r/PowerShell/comments/kj88m2/exporting_code_signing_certificates_from_an_exe/
+# - https://blog.codeinside.eu/2017/11/30/signing-with-signtool-dont-forget-the-timestamp/
+# - https://docs.microsoft.com/en-us/archive/blogs/ieinternals/everything-you-need-to-know-about-authenticode-code-signing
+#
+# This requires the SafeNet Client Application installed and running on the CI
+# box. Note that the USB token with the EV certificate is not visible when
+# logged in via RDP, only when you are logged in via AnyDesk/Chrome Desktop or
+# locally. Be sure to enable single logon in SafeNet so that you only have to
+# enter the password once.
```
Comment (timjarsky) [inline tools/sign-installer.sh:13]: @t-b mention that changes to this setting require a reboot?

**Review by timjarsky — CHANGES_REQUESTED** [review]

File: tools/sign-installer.sh (hunk 1 above)
Comment (t-b) [reply tools/sign-installer.sh]: @timjarsky Good point. I also revised the instructions to be better worded.

**Comment by t-b** [comment]
It's not working anymore, see http://bamboo.corp.alleninstitute.org/browse/MIES-PLAYIN464-BCI-12/log.

**Comment by t-b** [comment]
Signing works now. Merging!

**Pull request merged**